habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -800,10 +810,49 @@ def __init__(
     def recep_sets(self) -> Dict[str, ReceptacleSet]:
         return self._receptacle_sets
 
+    def init_scene_filters(
+        self, mm: habitat_sim.metadata.MetadataMediator, scene_handle: str
+    ) -> None:
+        """
+        Initialize the scene specific filter strings from metadata.
+        Looks for a filter file defined for the scene, loads filtered strings and adds them to the exclude list of all ReceptacleSets.
+
+        :param mm: The active MetadataMediator instance from which to load the filter data.
+        :param scene_handle: The handle of the currently instantiated scene.
+        """
+        scene_user_defined = mm.get_scene_user_defined(scene_handle)
+        filtered_unique_names = []
+        if scene_user_defined is not None and scene_user_defined.has_value(
+            "scene_filter_file"
```

Comment:
Are these strings documented somewhere?

Reply:
We are using the "user_defined" fields for this, so there is no official documentation. It is documented in the Receptacle filtering process and the tools used to generate the configs.
Once we have everything working for the dataset release we should add a doc page to the website about the entire Receptacle system.